modular_sand/code/game/objects/items/fleshlight.dm

```diff
@@ -689,11 +691,11 @@
 					to_chat(human, "<span class='warning'>The anus is covered or there is none!</span>")
 					return FALSE
 			if(CUM_TARGET_PENIS)
-				if(!human.has_penis(REQUIRE_EXPOSED))
+				if(!human.has_penis(REQUIRE_EXPOSED) && !human.has_strapon(REQUIRE_EXPOSED))
 					to_chat(human, "<span class='warning'>The penis is covered or there is none!</span>")
 					return FALSE
 			if(CUM_TARGET_URETHRA)
-				if(!human.has_penis(REQUIRE_EXPOSED))
+				if(!human.has_penis(REQUIRE_EXPOSED) && !human.has_strapon(REQUIRE_EXPOSED))
```

Comment:
You should most likely remove the strapon support on these until it's fully coded or else it'll runtime and cause errors whenever someone cums